### PR TITLE
[scss] textfield supports is-required on textfield-label

### PR DESCRIPTION
**Pull request by jeremie-lucca**
Supports is-required on textfield-label + demo

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://239-pr-lucca-front-LuccaSA.surge.sh

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://239-pr-lucca-front-LuccaSA.surge.sh

File: packages/scss/src/components/inputs/textfield/_textfield.scss [hunk 1]
```diff
@@ -223,13 +223,10 @@
...
 
 	&.is-required {
-		.textfield-label::after {
-			color: _color("error");
-			content: "*";
-			display: inline-block;
-			font-size: 1.2rem;
-			margin-left: .25em;
-			position: absolute;
+		.textfield-label {
```
Comment (sanlucca) [inline packages/scss/src/components/inputs/textfield/_textfield.scss]: You should just have .textfield-label;;after as there is no other imbricated style

**Comment by LuccaIntegration** [comment]
Travis automatic deployment: https://239-pr-lucca-front-LuccaSA.surge.sh

**Pull request merged**